src/goInstallTools.ts

```diff
@@ -26,15 +26,41 @@ let tools: { [key: string]: string } = {
 	guru: 'golang.org/x/tools/cmd/guru'
 };
 
-export function installTool(tool: string) {
-	outputChannel.clear();
+export function installTools(missing: string[]) {
 	outputChannel.show();
-	cp.exec('go get -u -v ' + tools[tool], { env: process.env }, (err, stdout, stderr) => {
-		outputChannel.append(stdout.toString());
-		outputChannel.append(stderr.toString());
-		if (err) {
-			outputChannel.append('exec error: ' + err);
+	outputChannel.clear();
+	outputChannel.appendLine('Installing ' + missing.length + ' missing tools');
+	missing.forEach((missingTool, index, missing) => {
+		outputChannel.appendLine('  ' + missingTool);
+	});
+
+	outputChannel.appendLine(''); // Blank line for spacing.
+
+	Promise.all(missing.map(tool => new Promise<string>((resolve, reject) => {
```

Comment:
One thing I was meaning to look into/change was running these serially instead of in parallel.  I'm pretty sure `go get` has issues when multiple gets are run in parallel, and users have reported that the existing extension occasionally runs into errors.  Since you are making changes here - may make sense to also just change this to do the gets serially